src/System.Net.NetworkInformation/tests/FunctionalTests/IPInterfacePropertiesTest_Linux.cs

```diff
@@ -0,0 +1,179 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Net.Sockets;
+using System.Net.Test.Common;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.Net.NetworkInformation.Tests
+{
+    [PlatformSpecific(PlatformID.Linux)]
+    public class IPInterfacePropertiesTest_Linux
+    {
+        private readonly ITestOutputHelper _log;
+
+        public IPInterfacePropertiesTest_Linux()
+        {
+            _log = TestLogging.GetInstance();
+        }
+
+        [Fact]
+        public void IPInfoTest_AccessAllProperties_NoErrors()
+        {
+            foreach (NetworkInterface nic in NetworkInterface.GetAllNetworkInterfaces())
```

Comment:
For collections we're iterating through, is there anything more we can assert?  For example, can we assert that the collection is non-empty?  Or will that not always be the case?